apps/artist/client/router.coffee

```diff
@@ -63,8 +64,8 @@ module.exports = class ArtistRouter extends Backbone.Router
   overview: ->
     @view = new OverviewView @options
     $('body').append @jump.$el
-    @view.on 'artist:overview:sync', =>
-      attachCTA @model
+    @view.on 'artist:overview:sync', (artist) =>
+      attachCTA new Artist(_.extend({}, artist, @model.attributes))
```

Comment:
This was needed b/c we want to use the initial bootstrapped server-side `@model`, but also 'decorate' it with the data coming back from the client-side fetch (namely, the related artist).